scripts/lint.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
you can save a line with `set -euxo pipefail`

Reply:
don't think this nit is worth rebuilding for. 😄